# This stores the data necessary for a time lapse map. The grids
# each store a map that is valid for a given time interval
TimeLapseOccGrid[] tmap